core/sdk-core/src/main/java/software/amazon/awssdk/core/protocol/VoidSdkResponse.java

```diff
@@ -31,7 +31,7 @@ public final class VoidSdkResponse extends SdkResponse {
 
     private static final List<SdkField<?>> SDK_FIELDS = Collections.unmodifiableList(Collections.emptyList());
 
-    private VoidSdkResponse(Builder builder) {
+    public VoidSdkResponse(Builder builder) {
```

Comment:
I wanted to invoke this class for usage in ProgressUpdater testing. I did see that this was ideal as it was a dummy implementation of SdkResponse. I could create a new implementation of this within the core class, I just wanted to avoid more duplication.